libcxx/test/std/numerics/rand/rand.dist/rand.dist.uni/rand.dist.uni.int/output.pass.cpp

```diff
@@ -0,0 +1,653 @@
+//===----------------------------------------------------------------------===//
```

Comment:
I think it's great that you added this test and documented it in your PR. That's the correct methodology for this kind of change. However, as you correctly mentioned in a comment, I think we shouldn't check-in the test since its value outside of this change isn't compelling.